chain/src/pipe.rs

```diff
@@ -53,7 +53,7 @@ pub fn process_block(b: &Block, mut ctx: BlockContext) -> Result<Option<Tip>, Er
 
 	info!(
```

Comment:
These can be changed to debug as well. There's a log for when the block is added (or errors when it fails) which should be enough.